src/cli/commands/home.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+
+import type {Reporter} from '../../reporters/index.js';
+import type Config from '../../config.js';
+import NpmRegistry from '../../registries/npm-registry.js';
+import parsePackageName from '../../util/parse-package-name.js';
+import * as fs from '../../util/fs.js';
+
+const opn = require('opn');
```

Comment:
Since I was copying from the info command, I'm not familiar with flow, and assumed there was some meaning to the difference.  Wiifix.